src/OrchardCore/OrchardCore.Email.Core/OrchardCore.Email.Core.csproj

```diff
@@ -9,6 +9,7 @@
       Core implementation of Email Module.
     </Description>
     <PackageTags>$(PackageTags) OrchardCoreFramework Email</PackageTags>
+    <RootNamespace>OrchardCore.Email</RootNamespace>
```

Comment:
Why is this and all the namespace changes necessary? It's confusing to have a different namespace than project name.

Reply:
restored